### DISCUSSION TITLE
Need help with Bricked device

### DISCUSSION THREAD

**Discussion by Xtqry** [community/community#137314] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Help I tried to flash twrp on my Samsung A30s using Odin. It came out as successfull on Odin but my phone isn't working. 
It turns on, shows the Samsung logo, the logo becomes green and it turns off. If anyone has any kind of information that could help me, please DM me.

**Comment by Mrsanyi123** [comment]
1. Boot into Recovery Mode:

- You can just power off your phone completely.
- Press and hold the Volume Up + Power Button + Home Button (if applicable) until the Samsung logo appears, then release the buttons.
- This should take you into TWRP recovery. If you can get into recovery, you can try Wiping Cache and Dalvik.

- Or else You Can just ask a professional to fix it for u!
 **Pls if this is helpful mark it as an answer!**

**Reply by Xtqry** [reply]
It didn't work, but I appreciate you taking your time to reply. I think I'll take it to a professional.

**Reply by Mrsanyi123** [reply]
Yea, that would be appreciated! Have a nice day!

**Comment by LiteBrite82** [comment]
Hi @Xtqry ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
